internal/dep/defaults.go

```diff
@@ -24,7 +24,7 @@ var (
 	DependencyEksctlSpot Dependency = &dependency{
 		name:               "eksctl-spot",
 		upstreamBinaryName: "eksctl",
-		version:            "0.77.0-6cfea8af",
+		version:            "0.124.0",
```

Comment:
@thorsteinnth You can update this to 0.127.0

Reply:
I will do a separate PR to update the k8s version and eksctl version